tools/rum-models-generator/Tests/rum-models-generator-coreTests/RUMModelsGeneratorTests.swift

```diff
@@ -21,12 +21,12 @@ final class RUMModelsGeneratorTests: XCTestCase {
 //
 //        let schemas = try RUMJSONSchemaFiles(folder: schemasFolderURL)
 //        let generator = RUMModelsGenerator()
-//
+
 //        print(">>>>>>>>>>>>>>>>>> Swift >>>>>>>>>>>>>>>>>>>>>>")
-//        print(try generator.printRUMModels(for: schemas, using: SwiftPrinter()))
+//        print(try generator.printRUMModels(for: schemas, using: .swift))
```

Comment:
💡 This test is commented out intentionally. It is used to run generator against real schemas from `rum-events-format` and do step-by-step analysis with debugger. This is annotated with test comment:
```swift
/// Test made for debugging purpose.
/// Uncomment it to run code generation for `../../../rum-events-format/schemas`.
```